students/rachel_s/session02/grid_print.py

```diff
@@ -0,0 +1,101 @@
+def print_grid_arbitrary():
+    """Prints a grid of arbitrary size."""
+    print("+ - - - + - - - + - - - +")
+    for i in range(3):
+        print("|       |       |       |")
+    print("+ - - - + - - - + - - - +")
+    for i in range(3):
+        print("|       |       |       |")
+    print("+ - - - + - - - + - - - +")
+    for i in range(3):
+        print("|       |       |       |")
+    print("+ - - - + - - - + - - - +")
+
+
+def print_grid(n):
+    """Prints a grid of width and height n that is 2 cells high and wide."""
+    width = n // 2
+    # Print top border
+    print("+", end=" ")
+    for i in range(width):
+        print("-", end=" ")
+    print("+", end=" ")
+    for i in range(width):
+        print("-", end=" ")
+    print("+")
+    # Print inside of cells
+    for j in range(width):
+        print("|", end=" ")
+        for i in range(width):
+            print(" ", end=" ")
+        print("|", end=" ")
+        for i in range(width):
+            print(" ", end=" ")
+        print("|")
+    # Print middle line
+    print("+", end=" ")
+    for i in range(width):
+        print("-", end=" ")
+    print("+", end=" ")
+    for i in range(width):
+        print("-", end=" ")
+    print("+")
+    # Print bottom insides of cells
+    for j in range(width):
+        print("|", end=" ")
+        for i in range(width):
+            print(" ", end=" ")
+        print("|", end=" ")
+        for i in range(width):
+            print(" ", end=" ")
+        print("|")
+    # Print bottom line
+    print("+", end=" ")
+    for i in range(width):
+        print("-", end=" ")
+    print("+", end=" ")
+    for i in range(width):
+        print("-", end=" ")
+    print("+")
+
+
+def print_grid2(grid_size, cell_size):
+    """Print a grid of a specified length and cell size
+
+    Keyword arguments:
+    grid_size: indicates the number of cells in the grid
+    cell_size: indicates the size of the cell"""
```

Comment:
Another really minor thing: It'd be better to clearly show that the docstring comments ends here by placing the closing `"""` in the next line at its beginning. Not sure if it's a requirement in docstring formatting, but I just found myself puzzled a bit about where the docstring ends (got so used to looking at the line beginnings only).